spec/features/site_customization/custom_pages_spec.rb

```diff
@@ -5,10 +5,9 @@
     scenario "See default content when custom page is not published" do
       custom_page = create(:site_customization_page,
         slug: "conditions",
-        title: "Custom conditions",
-        content: "New text for conditions page",
-        print_content_flag: true,
-        locale: "en"
+        title_en: "Custom conditions",
+        content_en: "New text for conditions page",
```

Comment:
Thanks to the beauty of globalize we can still write:

```ruby
      custom_page = create(:site_customization_page,
        slug: "conditions",
        title: "Custom conditions",
        content: "New text for conditions page",
        print_content_flag: true
      )
```
No need to add the `_en` prefix when creating records.

Reply:
The same idea applies to a few more specs changed in the pull request.

Reply:
I think is good to maintain uniformity with the prefixs ( much more easy for anyone that see the code for the first time). There are places in the this same spec where is needed to define this fields in differents languages and you need to use prefixs. But this my opinion, if you guys like without prefixs where is no needed I will changed. ;)